pages/install-device-page.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class InstallDevicePage {
```

Comment:
besides, we should always use the wording in the page as 99.999 % of the time will help us, page title is InstallationDevice and we also agree to avoid abbreviations as much as possible as they also tend to degenerate with time.

Reply:
I'm trying to be consistent in naming with what I see in other tests, but each test is different.